### PR TITLE
Multi-language support

### PR DESCRIPTION
**Pull request by BHess2653**
currently uses English(en), Spanish(es), and Alien(al).

.env example: LANGUAGE=en

messages are located in bot/locale/messages.json sorted by controllers.

### CODE REVIEW

**Comment by BHess2653** [comment]
Ready for Review and Release 3!

**Comment by brennondenny** [comment]
en, es, and al are all working with every command if it's in the scope add a check for the user's defined LANGUAGE in the .evn that it is an available option.

Other than that your good to merge! 👍

File: bot/controllers/channels.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const Command = require('../baseCommand.js');
 const util = require('apex-util');
 const msg = require('../locale/messages.json');
 
-const LANGUAGE = process.env.LANGUAGE;
+const lang = process.env.LANGUAGE;
```
Comment (mattpezzente) [inline bot/controllers/channels.js]: Make a provision for setting the default language to en, incase no language is set in the .env.

File: bot/controllers/verify.js [hunk 2]
```diff
@@ -110,9 +110,9 @@ class VerifyController extends BaseController {
...
 
       util.log('Code', code, 3);
-      return msg.verify.verifyEmailMsgStart[LANGUAGE] + ` ${(timeoutInMiliseconds / 1000) / 60} ` + msg.verify.verifyEmailMsgEnd[LANGUAGE];
+      return msg.verify.verifyEmailMsgStart[lang] + ` ${(timeoutInMiliseconds / 1000) / 60} ` + msg.verify.verifyEmailMsgEnd[lang];
     } else {
-      return msg.verify.verifyEmailDenied[LANGUAGE];
+      return msg.verify.verifyEmailDenied[lang];
     }
   }
 }
```
Comment (mattpezzente) [inline bot/controllers/verify.js]: Make sure to default all languages to en. If none are set, or if an incorrect langauge is set, all messages appear as *underfined* in Discord.

**Review by mattpezzente — CHANGES_REQUESTED** [review]
Make sure to implement the default fallback in-case the LANGUAGE is not set int he .env, or incase an incorrect LANGUAGE is set.

**Comment by mpeck99** [comment]
Besides the changes requested above overall looks good!

**Comment by BHess2653** [comment]
Changes have been made default Language is now en!

**Review by mattpezzente — APPROVED** [review]

**Comment by brennondenny** [comment]
Looks like CircleCI found some ESLint Issues.

**Pull request merged**